packages/metadata-common/src/MetadataEditor.tsx

```diff
@@ -14,25 +14,30 @@
  * limitations under the License.
  */
 
-import { FormGroup, Intent, ResizeSensor, Tooltip } from '@blueprintjs/core';
-
 import { MetadataService, IDictionary } from '@elyra/services';
-import { DropDown, RequestErrors } from '@elyra/ui-components';
+import { DropDown, RequestErrors, TextInput } from '@elyra/ui-components';
 
 import { ILabStatus } from '@jupyterlab/application';
 import { ReactWidget, showDialog, Dialog } from '@jupyterlab/apputils';
 import { CodeEditor, IEditorServices } from '@jupyterlab/codeeditor';
-import { InputGroup, Button } from '@jupyterlab/ui-components';
 
 import { find } from '@lumino/algorithm';
 import { IDisposable } from '@lumino/disposable';
 import { Message } from '@lumino/messaging';
+import {
+  InputLabel,
+  FormHelperText,
+  Button,
+  createMuiTheme,
+  ThemeProvider
+} from '@material-ui/core';
 
 import * as React from 'react';
 
 import { MetadataEditorTags } from './MetadataEditorTags';
 
 const ELYRA_METADATA_EDITOR_CLASS = 'elyra-metadataEditor';
+const ELYRA_METADATA_EDITOR_DARK = 'elyra-metadataEditor-dark';
```

Comment:
Should we append `class` for consistency on class constants? If so, please update the usages...
```suggestion
const ELYRA_METADATA_EDITOR_DARK_CLASS = 'elyra-metadataEditor-dark';
```